doc/develop/acq.rst

```diff
@@ -372,3 +372,70 @@ SPARC alignment:
 In mirror alignment mode, during background acquisition, the SEM is stopped to
 automatically activate the blanker (which will happen only if it's in the automatic setting).
 
+Optical Path Manager
+====================
+The goal of the Optical Path Manager is to the position actuators affecting the
+light path such that they are in the right configuration for a given acquisition type.
+It is found in ``acq.path.OpticalPathManager`` and is typically used as a singleton.
+The GUI takes care of instantiating it at init, and then it can be accessed via
+``MainGUIData.opm``. In practice, this is almost exclusively used on the SPARC.
+
+
+It has one important method:
+
+.. py:method:: setPath(mode, detector=None)
+
+   Sets all the necessary components of the optical path to the positions
+   required to achieve the requested mode.
+
+   :param path: The stream or the optical path mode
+   :type path: stream.Stream or str
+   :param detector:
+      When the `path` is a str, this allows
```

Comment:
I think the rule is:
allow + object + infinitive
or:
allow + gerund
So here `this allows defining`...